### PR TITLE
Feature/dx 1702 update about text for checkoutv3 implementations

### PR DESCRIPTION
**Pull request by trieutran91**
Update with text from https://www.figma.com/file/Ys5zmsVCmYt3yL4DruLzj0/FrontPage-V3-Entry?node-id=1199%3A69106

### CODE REVIEW

**Comment by trieutran91** [comment]
This branch is actually based on #1364. Which will avoid the need for rebase?

**Comment by asbjornu** [comment]
It's going to make a rebase simpler, but the merge-commit of #1364 is going to move `develop` ahead so it will have to be merged into this PR, or this PR will have to be rebased on `develop`. I prefer the latter (rebase).

**Comment by trieutran91** [comment]
> It's going to make a rebase simpler, but the merge-commit of #1364 is going to move `develop` ahead so it will have to be merged into this PR, or this PR will have to be rebased on `develop`. I prefer the latter (rebase).

Noted! Rebase it is :)

File: checkout-v3/index.md [hunk 1]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
 
-When building your checkout page you have two main paths. Either you let us
-provide you with the entire checkout solution. This includes payer
-identification and payment menu, where your customer themselves can choose how
-to pay. Or, you can choose to only use our payment option (also called the
-payment menu), where you are in charge of collecting and storing the customer
-data.
+Here, you can choose from two main
+paths. Either you let us provide you with a full checkout solution (including
+both consumer identification and payment menu) - meaning the consumer self can
```
Comment (arebra) [inline checkout-v3/index.md:85]:
```suggestion
both consumer identification and payment menu) - meaning the payer themself can
```

File: checkout-v3/index.md [hunk 2]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
-
--   **Authentication:** The process for verifying the Payer's identity via
-    Strong Consumer Authentication (e.g. BankID).
+-   **Authentication:** The process for verifying the payers identity.
 
 -   **Delivery Info:** Information about where the goods should be delivered.
 
--   **PSP:** The service of providing payment methods in the checkout or payment
-    menu.
+-   **Payer Info:** The consumers’ personal data e.g. name, address, card number
```
Comment (arebra) [inline checkout-v3/index.md:98]:
```suggestion
-   **Payer Info:** The payer’s personal data e.g. name, address, card number
```

File: checkout-v3/index.md [hunk 3]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+    etc.
+
+-   **PSP:** The service of providing payment methods in the checkout.
```
Comment (arebra) [inline checkout-v3/index.md:101]:
```suggestion
-   **PSP:** The service of providing payment instruments in the checkout.
```

File: checkout-v3/index.md [hunk 4]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+only. This means that you are in charge of collecting and storing the consumer
+data and the payment methods to be presented.
 
-Regardless if you choose our checkout or payments only option, you will always
-be able to decide which payment options are available for the payer. Hence
-you can show all available payment options for that specific market, or just one
-or two if that makes more sense for your business.
+Regardless of whether you choose
+to go with the full checkout or payments only, we will supply you with a variety
+of payment methods and features to cater for your business needs.
```
Comment (arebra) [inline checkout-v3/index.md:92]:
```suggestion
of payment instruments and features to cater for your business needs.
```

File: checkout-v3/index.md [hunk 5]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
 
 {% capture tab1_intro %}
 
 ## Full Checkout
 
-By using the **Full Checkout**, we help you collect and safely store payer
-data. We can also prefill payer info in the checkout if they have agreed to
-let us store their info. All of our implementations support both single payment
-options and the full payment offering.
+From the **Full Checkout**, you will get access to a number of payment methods
```
Comment (arebra) [inline checkout-v3/index.md:109]:
```suggestion
From the **Full Checkout**, you will get access to a number of payment instruments.
```

File: checkout-v3/index.md [hunk 6]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+selected to support the needs of the local market. We help you collect and
+safely store the consumers' data. And if the consumer agrees, we will store
```
Comment (arebra) [inline checkout-v3/index.md:111]:
```suggestion
safely store the payer's data. And if the payer agrees, we will store
```

File: checkout-v3/index.md [hunk 7]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+their information to have it prefilled next time they shop.
```
Comment (arebra) [inline checkout-v3/index.md:112]:
```suggestion
their information to have it prefilled the next time they shop.
```

File: checkout-v3/index.md [hunk 8]
```diff
@@ -123,7 +120,10 @@ options and the full payment offering.
   icon_content='shopping_cart'
   icon_outlined=true
   button_content='Proceed'
-  text='We collect and verify the identity of your payer. We also collect the billing and shipping address, and we store the payer information. With our PSP you are always able to choose one or more payment methods.'
+  text='The ultimate implementation pack for anyone new to this, needing support
```
Comment (arebra) [inline checkout-v3/index.md:123]:
```suggestion
  text='The ultimate implementation pack for anyone new to this, who needs support
```

File: checkout-v3/index.md [hunk 9]
```diff
@@ -123,7 +120,10 @@ options and the full payment offering.
...
+  with all parts covering the checkout. We collect and verify the consumer
```
Comment (arebra) [inline checkout-v3/index.md:124]:
```suggestion
  with all parts covering the checkout. We collect and verify the payer's
```

File: checkout-v3/index.md [hunk 10]
```diff
@@ -123,7 +120,10 @@ options and the full payment offering.
...
+  identity, the billing and shipping address and store the consumer data.
```
Comment (arebra) [inline checkout-v3/index.md:125]:
```suggestion
  identity, billing and shipping address, and store the payer data.
```

File: checkout-v3/index.md [hunk 11]
```diff
@@ -136,10 +136,10 @@ options and the full payment offering.
   icon_content='shopping_cart'
   icon_outlined=true
   button_content='Proceed'
-  text="The payer data you have collected, is shared with us for verification and
-  storage. If you have a login, your payer won't need to fill out their
-  information twice. With our PSP you are always able to choose one or more
-  payment methods."
+  text="On the next level on your growth journey, looking for someone to
+  authenticate and store the consumer data you collected? We got you covered and
```
Comment (arebra) [inline checkout-v3/index.md:140]:
```suggestion
  authenticate and store the payer data you collected? We've got you covered, so the
```

File: checkout-v3/index.md [hunk 12]
```diff
@@ -152,11 +152,10 @@ options and the full payment offering.
   icon_content='shopping_cart'
   icon_outlined=true
   button_content='Proceed'
-  text="In order to implement this solution, you need to be able to both collect and
-  verify your payer's data using SCA. We store the payer information, and
-  with our PSP you are always able to choose one or more payment methods."
-  table_content=page.table_content_enterprise
-  header=page.header
+  text="If you only need support with data storage and payment methods - this is
```
Comment (arebra) [inline checkout-v3/index.md:155]:
```suggestion
  text="If you only need support with data storage and payment instruments - this is
```

File: checkout-v3/index.md [hunk 13]
```diff
@@ -136,10 +136,10 @@ options and the full payment offering.
   icon_content='shopping_cart'
   icon_outlined=true
   button_content='Proceed'
-  text="The payer data you have collected, is shared with us for verification and
-  storage. If you have a login, your payer won't need to fill out their
-  information twice. With our PSP you are always able to choose one or more
-  payment methods."
+  text="On the next level on your growth journey, looking for someone to
+  authenticate and store the consumer data you collected? We got you covered and
+  the consumer doesn’t need to fill out their information twice. Simply
```
Comment (arebra) [inline checkout-v3/index.md:141]:
```suggestion
payer won't have to fill out their information twice. Simply
```

File: checkout-v3/index.md [hunk 14]
```diff
@@ -167,10 +166,9 @@ options and the full payment offering.
 
 ## Payments Only
 
-If you are looking for our payments package, you will have the flexibility to
-build your own checkout flow. You collect the payer data and own the entire
-checkout process. The **Payments Only** implementation supports both single
-payment options and the full payment offering.
+With our payments only package, you collect the consumer data and have the
```
Comment (arebra) [inline checkout-v3/index.md:169]:
```suggestion
With our payments only package, you collect the payer data and have the
```

File: checkout-v3/index.md [hunk 15]
```diff
@@ -167,10 +166,9 @@ options and the full payment offering.
...
+flexibility to build your own checkout flow. This implementation supports our
+full range of payment methods.
```
Comment (arebra) [inline checkout-v3/index.md:171]:
```suggestion
full range of payment instruments.
```

File: checkout-v3/index.md [hunk 16]
```diff
@@ -181,9 +179,9 @@ payment options and the full payment offering.
   icon_content='shopping_cart'
   icon_outlined=true
   button_content='Proceed'
-  text="In order to implement this solution, you need to be able to both collect and
-  verify your payer's data using SCA. You also store the payer information.
-  With our PSP you are always able to choose one or more payment methods."
+  text="If you can collect, verify and store your consumer data, and if needed -
```
Comment (arebra) [inline checkout-v3/index.md:182]:
```suggestion
  text="If you can collect, verify and store your payer data, and if needed -
```

File: checkout-v3/index.md [hunk 17]
```diff
@@ -181,9 +179,9 @@ payment options and the full payment offering.
...
+  the delivery address - this is the option for you. With our PSP you can decide
+  if you want to offer a single payment method or the full payment offering. "
```
Comment (arebra) [inline checkout-v3/index.md:184]:
```suggestion
  if you want to offer a single payment instrument or the full payment offering."
```

File: index.md [hunk 18]
```diff
@@ -3,7 +3,7 @@ title: Developer Portal
 layout: front-page
 front_page:
   ingress: |
-    Here you'll find the full toolbox for how to integrate our payment solutions and acquaint yourself with their different features and functionalities.
+    Here you'll find the full toolbox for how to integrate our payment solutions and acquaint yourself with their different features and functionalities and learn how to integrate them.
```
Comment (arebra) [inline index.md:6]:
```suggestion
    Here you'll find the full toolbox for how to integrate our payment solutions, acquaint yourself with their different features and functionalities, and learn how to integrate them.
```

**Review by arebra — CHANGES_REQUESTED** [review]
The language geek back at it.

File: checkout-v3/index.md [hunk 19]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+goods, where that isn't needed. But this is also a matter of which data you can
+and/or wish to collect and manage yourself.
 
-When building your checkout page you have two main paths. Either you let us
-provide you with the entire checkout solution. This includes payer
-identification and payment menu, where your customer themselves can choose how
-to pay. Or, you can choose to only use our payment option (also called the
-payment menu), where you are in charge of collecting and storing the customer
-data.
+Here, you can choose from two main
```
Comment (arebra) [inline checkout-v3/index.md:83]:
```suggestion
Here, you can choose between two main
```

File: checkout-v3/index.md [hunk 20]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+paths. Either you let us provide you with a full checkout solution (including
+both consumer identification and payment menu) - meaning the payer themself can
```
Comment (arebra) [inline checkout-v3/index.md:85]:
```suggestion
both payer identification and payment menu) - meaning the payer themselves can
```

File: checkout-v3/index.md [hunk 21]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+choose how they'd like to pay. Or, you can decide to use the option for payments
+only. This means that you are in charge of collecting and storing the consumer
```
Comment (arebra) [inline checkout-v3/index.md:87]:
```suggestion
only. This means that you are in charge of collecting and storing the payer
```

File: checkout-v3/index.md [hunk 22]
```diff
@@ -73,46 +74,42 @@ table_content_payments:
...
+data and the payment methods to be presented.
```
Comment (arebra) [inline checkout-v3/index.md:88]:
```suggestion
data and the payment instruments to be presented.
```

**Review by arebra — CHANGES_REQUESTED** [review]
I missed a couple!

**Review by arebra — COMMENTED** [review]
Great :D

**Review by arebra — APPROVED** [review]
Wrong radio button :)

**Review by asbjornu — APPROVED** [review]

**Pull request merged**